frontend/src/components/tabs/MetricsTab.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2021 The Kubeflow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Artifact, ArtifactType, Execution } from '@kubeflow/frontend';
+import HelpIcon from '@material-ui/icons/Help';
+import * as React from 'react';
+import { useQuery } from 'react-query';
+import { Link } from 'react-router-dom';
+import IconWithTooltip from 'src/atoms/IconWithTooltip';
+import { color, commonCss, padding } from 'src/Css';
+import { ExecutionHelpers, getOutputArtifactsInExecution } from 'src/lib/MlmdUtils';
+import { filterArtifactsByType, getArtifactTypes } from 'src/lib/OutputArtifactLoader';
+import Banner from '../Banner';
+import { RoutePageFactory } from '../Router';
+import ROCCurve, { DisplayPoint, ROCCurveConfig } from '../viewers/ROCCurve';
+
+const ROC_CURVE_DEFINITION =
+  'The receiver operating characteristic (ROC) curve shows the trade-off between true positive rate and false positive rate. ' +
+  'A lower threshold results in a higher true positive rate (and a higher false positive rate), ' +
+  'while a higher threshold results in a lower true positive rate (and a lower false positive rate)';
+
+type ConfidenceMetrics = {
```

Comment:
nit: can we name this `ConfidenceMetric`? does this need to be plural?

Reply:
Makes total sense, updating to singular.